src/BokutachiHook.cpp

```diff
@@ -362,9 +366,10 @@ void BokutachiHook::Init() {
 	std::println("[BokutachiHook] Init Done.");
 	std::fflush(stdout);
 	CheckTachiApi();
-	CheckVersion();
+	if (checkUpdates)
+		CheckVersion();
```

Comment:
```suggestion
	if (checkUpdates) CheckVersion();
```
This is my preferred style.